src/main.js

```diff
@@ -5,10 +5,25 @@ import './assets/main.css';
 
 import plausible from './plugins/plausible';
 
+const clickOutside = {
+  beforeMount: (el, binding) => {
+    el.clickOutsideEvent = (event) => {
+      if (!(el == event.target || el.contains(event.target))) {
+        binding.value();
+      }
+    };
+    document.addEventListener('click', el.clickOutsideEvent);
```

Comment:
Can you change the `click` event to `mousedown`? That way the color picker wont close if the user first clicks inside the picker and then drags and releases the click outside of it. Thanks!

Reply:
Sure 💯